runtime/src/accounts_db.rs

```diff
@@ -8246,69 +8227,6 @@ pub mod tests {
         }
     }
 
-    #[test]
-    fn test_account_balance_for_capitalization_normal() {
-        // system accounts
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(10, &Pubkey::default(), false),
-            10
-        );
-        // any random program data accounts
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(
-                10,
-                &solana_sdk::pubkey::new_rand(),
-                false,
-            ),
-            10
-        );
-    }
-
-    #[test]
-    fn test_account_balance_for_capitalization_sysvar() {
-        let normal_sysvar = solana_sdk::account::create_account_for_test(
-            &solana_sdk::slot_history::SlotHistory::default(),
-        );
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(
-                normal_sysvar.lamports,
```

Comment:
I don't know if elsewhere we test normal_sysvar.lamports. You should probably replace the function call with normal_sysvar.lamports and leave the assert. The next one where we pass the 10 is obviously now useless.

Reply:
Good catch - I agree there is value to keeping the first half her in `test_account_balance_for_capitalization_sysvar()` and `test_account_balance_for_capitalization_native_program`